dev/PushNotifications/PushManager.h

```diff
@@ -11,6 +14,12 @@ namespace winrt::Microsoft::ProjectReunion::implementation
         PushManager() = default;
 
         static Windows::Foundation::IAsyncOperationWithProgress<Microsoft::ProjectReunion::ChannelResult, Microsoft::ProjectReunion::ChannelResult> CreateChannelAsync(winrt::guid remoteId);
+
+    private:
+        static bool isChannelRequestRetryable(const winrt::hresult& hrException);
+        static std::list<winrt::guid> remoteIdList;
+        static std::mutex mtx;
```

Comment:
```suggestion
        static std::mutex mutex;
```
Nit: Seems unnecessary to shorten this and not all the other variables.

Reply:
Looks like we need to get the naming conventions published here.  For instance, statics should have an `s_` prefix.  Just straight `mutex` isn't great.

Also, not required, but consider `std::vector<winrt::guid>` to reduce memory allocation fragmentation.  I suspect this list is almost always less than 3-4 values right now.

Reply:
Sure will address this.